### PR TITLE
feat(KSM-4791, nav.js, blog.js, home.js, count.js):

### PR DESCRIPTION
**Pull request by Jmg047**
Progress check: I plan to place the 'show' button in the body outside of the navbar component, though for now I'm having some trouble with the height of my buttons, ideally they ought to be max-hight of the navbar (note: this is subjective and doesn't have to be a mandatory implementation though). Any Feedback is appreciated!

### CODE REVIEW

**Review by pamelagilmour — COMMENTED** [review]
Looks good so far! 
+ As a visual learner, one trick I learned from Jason is to add a border to every element, or at least the elements you are style debugging and their container(s) etc. It helps to visualized where the issue lies. Also have you tried using the DevTools? It can also be another way to debug styles.

+ I noticed your style sheet has some inconsistent spacing which can sometimes lend to styling bugs. I don't think its causing an issue currently but it's good to double check.

+ I noticed you are using flex in your nav bar. I think that's where the styling mismatch you are having lies. I think it might be with lines 90 and 91 in App.scss and/or putting each section of your nav bar in a " inner container" or <div>. There are many ways to move forward. Also setting a width for your buttons....LMK if you want to pair and I can show you all that stuff 👆 in action.

**Comment by Jmg047** [comment]
React Routes have been implemented for different pages with different content. There is a navbar with buttons that go to different routes and a form. Each route has different content while the navbar stays constant at the top of each page.

``Testing:``
Navigate to ``home``, ``counter``, ``about``, & ``contact``
mess around with screen size for the mobile responsiveness

File: client/src/App.scss [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+      cursor: pointer;
+      transition: background-color 0.3s ease;
+
+      &:hover {
+        background-color: gray;
+      }
+    }
+  }
+
+  .navbar {
```
Comment (pamelagilmour) [inline client/src/App.scss:148]: It looks like maybe you are repeating styles, maybe?

File: client/src/component/nav.js
Comment (pamelagilmour) [inline client/src/component/nav.js:1]: Congrats on this.

**Review by pamelagilmour — CHANGES_REQUESTED** [review]
Okay looks good so far! Double check your stylesheet.

**Comment by pamelagilmour** [comment]
I just left a review, but I forgot to check mobile. The mobile needs a little bit of work, much of the page is off screen. I'm not sure if that is what you are intending or not.

**Comment by Jmg047** [comment]
Good catch! I definitely didn't intend for everything to be off screen on mobile, I'll go ahead and fix that and make another PR

**Review by pamelagilmour — APPROVED** [review]
LGTM!!! Great job!

**Pull request merged**